frontend/javascripts/oxalis/controller/td_controller.js

```diff
@@ -260,10 +269,6 @@ class TDController extends React.PureComponent<Props> {
   }
 
   render() {
-    if (!this.controls) {
-      return null;
-    }
```

Comment:
By postponing this null-check to the exact points where it's necessary (i.e., in `setTargetAndFixPosition`), we can eagerly instantiate the CameraController which should fix the issue.